### DISCUSSION TITLE
Display colored dots for heroes on the minimap [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4587] [Ideas]
In the original game, the player's heroes are indicated by colored dots on the minimap (the opponent's heroes are not).
In fheroes2, this is not the case.
Also, I think it would be nice to display colored dots for the opponent heroes on the minimap.

In original game:

![Colored dots for heroes 001](https://user-images.githubusercontent.com/43583994/140809147-2ea46db6-c3e1-40d6-ae13-f9f2f0b69aab.png)

In fheroes2:

![Colored dots for heroes 002](https://user-images.githubusercontent.com/43583994/140809200-dad0d09b-d080-4a72-9bd7-f2a9016b7617.png)

![Colored dots for heroes 003](https://user-images.githubusercontent.com/43583994/140809218-46f609dc-0311-4be9-843a-2c032d4138e8.png)

![Colored dots for heroes 004](https://user-images.githubusercontent.com/43583994/140809226-7e24e11c-d66b-4de1-8f2a-d157dd3cbf93.png)

**Comment by ihhub** [comment]
Hi @LeHerosInconnu , this is definitely a bug. I'll create an issue based on this discussion.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
